src/tool/subcommands/api_cmd/test_snapshot.rs

```diff
@@ -24,12 +24,21 @@ use serde::{Deserialize, Serialize};
 use std::{path::Path, sync::Arc};
 use tokio::{sync::mpsc, task::JoinSet};
 
+#[derive(Default, Hash, Eq, PartialEq, Debug, Clone, Serialize, Deserialize)]
+pub struct Payload(#[serde(with = "crate::lotus_json::base64_standard")] pub Vec<u8>);
+
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct Index {
+    pub eth_mappings: std::collections::HashMap<String, Payload>,
+}
+
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct RpcTestSnapshot {
     pub chain: NetworkChain,
     pub name: String,
     pub params: serde_json::Value,
     pub response: Result<serde_json::Value, String>,
+    pub index: Index,
```

Comment:
Let's make new fields optional to maintain backward compatibility with old test snapshots

Reply:
Good point!